src/components/start-process/start-process.tsx

```diff
@@ -0,0 +1,81 @@
+import { Link as RouterLink, useHistory } from "react-router-dom";
+
+import { Form, Formik } from "formik";
+import * as Yup from "yup";
+
+import { locale } from "../../services";
+
+import { Button, Checkbox, Link } from "@mtfh/common/lib/components";
+
+import "./styles.scss";
+
+const { components } = locale;
+
+interface StartProcessProps {
+  process: {
+    thirdPartyCondition: string;
+    ThirdPartyContent: any;
+    riskHeading: string;
+    RiskContent: any;
+  };
+  backLink: string;
+  processName: string;
+}
+
+export const schema = Yup.object({
+  condition: Yup.bool().oneOf([true]),
+});
+
+export type FormData = Yup.Asserts<typeof schema>;
+
+export const StartProcess = ({ processName, process, backLink }: StartProcessProps) => {
+  const history = useHistory();
+  const { thirdPartyCondition, ThirdPartyContent, riskHeading, RiskContent } = process;
+  const hasThirdPartyContent = !!thirdPartyCondition;
+
+  return (
+    <Formik<FormData>
+      initialValues={{
+        condition: !hasThirdPartyContent,
+      }}
+      validateOnChange
+      validateOnMount
+      validateOnBlur={false}
+      validationSchema={schema}
+      onSubmit={() => {
+        history.push(`/processes/${processName}/generated-id`);
+      }}
+    >
+      {(properties) => (
+        <Form>
+          {hasThirdPartyContent && (
+            <>
+              <h3>{components.startProcess.thirdPartyHeading}</h3>
+              <ThirdPartyContent />
+              <Checkbox onChange={properties.handleChange} id="condition">
```

Comment:
`<Checkbox name="condition" id="condition">` should work the same

Reply:
What do you mean by this?

Reply:
Sorry that wasn't clear at all. You can also use the Field or InlineField to wrap this component without having to pass the on change handler. As it will hook into the FormikContext based on the name of the input.

Reply:
Thanks 👍